src/components/Instructions.component.js

```diff
@@ -5,14 +5,29 @@ import './../styles/instructions.css'
 
 class Instructions extends Component {
   render() {
+    var myitems = this.props.items.filter(function(item) {
+      return item.length > 2
+    })
+
+    myitems = myitems.map(function(item) {
```

Comment:
1) don't use `var`
2) you might be tempted to use `let` since you are reassigning `myitems` here, but a better way is to chain the `filter` and `map`s and assign the result of all the transformations to a variable:
```javascript
const myItems = this.props.items.filter(item => item.length > 2)
  .map(item => ...)
  .filter(item => ...)
  .map(blah => blah blah)